web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingsinformasjonJson.kt

```diff
@@ -272,11 +281,17 @@ data class PersonligOppmøteJson(
 )
 
 data class BosituasjonJson(
-    val delerBolig: Boolean,
-    val delerBoligMed: String?,
-    val ektemakeEllerSamboerUnder67År: Boolean?,
+    val epsFnr: Fnr?,
+    val delerBolig: Boolean?,
     val ektemakeEllerSamboerUførFlyktning: Boolean?,
     val begrunnelse: String?
 )
 
-data class EktefelleJson(val fnr: Fnr?)
+data class EktefelleJson(
```

Comment:
Er det ikke bedre at EktefelleJson-objektet kan være null i seg selv, istedet for at alle feltene kan være null?

Reply:
EktefelleJson er allerede Nullable. Problemet kommer fra(?) at vi alltid oppdaterer 1 og 1 vilkår om gangen. (1 vilkår har data, vi oppdaterer. Alle andre gjør vi ingenting med). Ektefelle er lagt til som en del av behandlingsinfo. 

Hvis jeg som saksbehandler, legger in EPS i formue. går til sats steget, og sier "å nei shit, han har ikke EPS". går til formue igjen, sier nei på EPS. og går videre, så vil ikke den infoen ha oppdatert seg fordi vi da sender den som null. Fikset var å gjøre alle feltene inne i den nullable, og alltid i det minste sende et object med null verdier for å trigge endringen. 

Det andre fikset kan se på at siden vi legger EPS i formue, så skal den være en del av formue-json. Følte ikke om jeg skulle gjøre et så inngripende tiltak eller ikke.

Reply:
ok, jeg tror jeg forstår. Siden vi har egne typer for json, har du mulighet til å la domenet være uendret, men legge på nye json-objekter, hvis det gir mer mening for frontend.